pkgs/applications/misc/playonlinux/default.nix

```diff
@@ -76,7 +78,7 @@ in stdenv.mkDerivation {
     ./0001-fix-locale.patch
   ];
 
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [ makeWrapper glib wrapGAppsHook ];
```

Comment:
```suggestion
  nativeBuildInputs = [ makeWrapper wrapGAppsHook ];
```
 `glib` should go in `buildInputs` otherwise cross compilation breaks.

Reply:
I was not aware of this, thanks! It should be fine now.